create.js

```diff
@@ -0,0 +1,125 @@
+import Completed from './module/completed';
+import Clear from './module/clear';
+
+export default function create() {
+  const addListInput = document.querySelector('.add');
+  const AddToList = document.querySelector('.add-to-list');
```

Comment:
I think it is better if you change the AddToList constant to camel case notation in order to follow JavaScript best practices.